squarelet/organizations/serializers.py

```diff
@@ -262,20 +264,22 @@ class Meta:
 
 
 class PressPassEntitlmentSerializer(serializers.ModelSerializer):
-    client = serializers.SerializerMethodField()
+    client_data = serializers.SerializerMethodField()
```

Comment:
https://www.django-rest-framework.org/api-guide/relations/#nested-relationships

This is better represented as:
```
client = PressPassClientSerializer(read_only=True)
```
I believe.  Also, why did you separate `client` and `client_data`?